recipes/diligent-core/all/conanfile.py

```diff
@@ -126,42 +143,22 @@ def _diligent_platform(self):
         elif self.settings.os == "watchOS":
             return "PLATFORM_TVOS"
 
-    def _configure_cmake(self):
-        if self._cmake:
-            return self._cmake
-        self._cmake = CMake(self)
-        self._cmake.definitions["DILIGENT_BUILD_SAMPLES"] = False
-        self._cmake.definitions["DILIGENT_NO_FORMAT_VALIDATION"] = True
-        self._cmake.definitions["DILIGENT_BUILD_TESTS"] = False
-        self._cmake.definitions["DILIGENT_NO_DXC"] = True
-        self._cmake.definitions["DILIGENT_NO_GLSLANG"] = not self.options.with_glslang
-        self._cmake.definitions["SPIRV_CROSS_NAMESPACE_OVERRIDE"] = self.options["spirv-cross"].namespace
-        self._cmake.definitions["BUILD_SHARED_LIBS"] = False
-        self._cmake.definitions["DILIGENT_CLANG_COMPILE_OPTIONS"] = ""
-        self._cmake.definitions["DILIGENT_MSVC_COMPILE_OPTIONS"] = ""
-
-        self._cmake.definitions["ENABLE_RTTI"] = True
-        self._cmake.definitions["ENABLE_EXCEPTIONS"] = True
-
-        self._cmake.definitions[self._diligent_platform()] = True
-        self._cmake.configure(build_folder=self._build_subfolder)
-        return self._cmake
-
     def build(self):
         self._patch_sources()
-        cmake = self._configure_cmake()
+        cmake = CMake(self)
+        cmake.configure()
```

Comment:
Is this taking the root CMAKELISTS.TXT that's being exported?

Reply:
No, there is no CmakeLists.txt from export at all. There is only original CMakeLists.txt from diligent-core

Reply:
This comment is obsolete, I made it using CMakeLists from the recipe folder since then. It's much more convenient comparing to apply changes to all patches one-by-one